persistence-modules/read-only-transactions/src/main/java/com/baeldung/read_only_transactions/mysql/spring/entities/TransactionEntity.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.read_only_transactions.mysql.spring.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "transactions")
+public class TransactionEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String uuid;
+
+    public TransactionEntity() {}
```

Comment:
Since you only have this constructor, I think you can remove it and let the Java translator create it for you. Otherwise, add a comment.